.github/stale.yml

```diff
@@ -0,0 +1,18 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 7
```

Comment:
I think we want the first "ping" (becoming stale) to be earlier, and the final closure to be later.
```suggestion
daysUntilStale: 30
# Number of days of inactivity before a stale issue is closed
daysUntilClose: 60
```

Reply:
I think 30 days until stay is way too short for the current state and respecting @vyasr wish to be conservative about this. And then 60 days until actually closing it is way too long. How about: 60-14 ?

Reply:
I think 14 days should be more than enough time for anybody involved to respond to this and "prevent" the close. Also keep in mind that we can re-open an issue if needed, it's not like they get deleted.

Reply:
I agree with @csadorf here, the time between when an issue is marked stale and when it's closed should be relatively short. 60-14 seems like a reasonable compromise. A total of 2.5 months of inactivity is a reasonable limit for when to close.

I agree that we can re-open issues in theory, but my concern has always been that this happens infrequently in practice. Github's search defaults to open issues, and unless people know to search closed ones (or have enough institutional knowledge to remember old closed issues) it's hard to remember. The overhead of open issues is definitely big enough to make this PR worthwhile, just pointing out why I've always been a bit hesitant to rely on the fact that closed issues don't vanish.

Reply:
> I agree with @csadorf here, the time between when an issue is marked stale and when it's closed should be relatively short. 60-14 seems like a reasonable compromise. A total of 2.5 months of inactivity is a reasonable limit for when to close.

Ok, would you prefer to have a prolonged discussion about this then? Otherwise I'd request you approve the PR.

> I agree that we can re-open issues in theory, but my concern has always been that this happens infrequently in practice. Github's search defaults to open issues, and unless people know to search closed ones (or have enough institutional knowledge to remember old closed issues) it's hard to remember. The overhead of open issues is definitely big enough to make this PR worthwhile, just pointing out why I've always been a bit hesitant to rely on the fact that closed issues don't vanish.

I don't think this is a problem. Either a particular issue crops up again and is important enough that there is either some institutional knowledge about it or that people go through the "effort" of also searching closed issues [1] or it is not that important and there simply is a little bit of duplication.

[1] I honestly don't understand that. Whenever I don't find what I'm looking for among the open issues, I of course also search the closed issues to check whether the issue has come up before or has already been resolved. I've certainly encountered auto-closed (stale) issues in this way on which I then simply commented and asked for re-opening.

Reply:
Then you're better than me :) That wasn't true for me until maybe a couple of years ago, certainly not during the first few years of my dev life. Unless I knew that an issue previously existed and was closed, I wouldn't have thought to search through closed issues even though I knew it was a possibility. And I certainly see enough duplicate issues (usually of closed issues, but also occasionally of open ones) that this is definitely not a problem that's restricted to me.